signals/src/signal.rs

```diff
@@ -450,6 +458,109 @@ impl<A> Signal for Flatten<A>
     }
 }
 
+#[doc(hidden)]
```

Comment:
Why is this hidden?

Reply:
Isn't the Broadcast struct itself an implementation detail (at some point once impl Trait lands the `.broadcast()` method could just return `impl Signal<Item=Self::Item> + Clone`)?

Reply:
Yes, but it'll still be a while before `impl` is in Stable, so for now it is a part of the API.

Users who use `.broadcast()` will need to write out the `Broadcast` type, so hiding it away is quite counter-productive.